File: Hoteles-Reserva-WS/src/java/models/Meal.java
package models;

import java.util.List;

public class Meal {
    private int idMeal;
    private String name, description;
    private double cost;
    private Boolean active;
    private List<Food> lstFood;

    public int getIdMeal() {
        return idMeal;
    }

    public void setIdMeal(int idMeal) {
        this.idMeal = idMeal;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public double getCost() {
        return cost;
    }

    public void setCost(double cost) {
        this.cost = cost;
    }

    public Boolean getActive() {
        return active;
    }

    public void setActive(Boolean active) {
        this.active = active;
    }

    public List<Food> getLstFood() {
        return lstFood;
    }

    public void setLstFood(List<Food> lstFood) {
        this.lstFood = lstFood;
    }
    
}

File: Hotel-Reservacion/Hotel-Reservacion/Models/Meal.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Hotel_Reservacion.Models
{
    public class Meal
    {
        private int idMeal;
        private String name, description;
        private double cost;
        private Boolean active;

        public int IdMeal
        {
            get
            {
                return idMeal;
            }
            set
            {
                idMeal = value;
            }
        }
        public String Name
        {
            get
            {
                return name;
            }
            set
            {
                name = value;
            }
        }
        public String Description
        {
            get
            {
                return description;
            }
            set
            {
                description = value;
            }
        }
        public Double Cost
        {
            get
            {
                return cost;
            }
            set
            {
                cost = value;
            }
        }
        public bool Active
        {
            get
            {
                return active;
            }
            set
            {
                active = value;
            }
        }
    }
}